content/pages/kits.md

```diff
@@ -0,0 +1,72 @@
+Title: Ordering, Building, and Operating DIYnamics Kits
+Slug: kits
+
+## The DIYnamics Kits
+
+The DIYnamics Project provides the resources for you to order, build and operate your own rotating tank platform! As detailed in the [About](/pages/about.html) page, these devices are used to model geophysical fluid dynamics. Rotating tanks of water feature fluid behavior analogous to planetary fluid bodies fundamentally impacted by rotation. As part of our Do-It-Yourself approach, our goal is for educators to perform experiments with rotating tank platforms themselves. Hands-on demonstrations provide a valuable opportunity for students to ask questions and draw conclusions in linking experimental results to real world phenomena.
```

Comment:
This is all good material, but I think the less text the better here.  We don't want to overwhelm people, and odds are if they got to this page they won't need so much background.  So I would just delete the "As detailed...", "Rotating tanks...", and "As part of..." sentences but keep the other two.